app/scripts/models/account.js

```diff
@@ -51,7 +51,6 @@ define(function (require, exports, module) {
     customizeSync: undefined,
     declinedSyncEngines: undefined,
     keyFetchToken: undefined,
-    password: undefined,
```

Comment:
I should probably put a note at the top of this file about handling sensitive data and why password is not part of the model.